### PR TITLE
Unit 3 mini projects

### PR DESCRIPTION
**Pull request by ecwalsh**
This is just the mini-project READMEs for what's currently in the roadmap. 

I think we should consider adding more React content. The OpenClassrooms guide suggests we only have around 10 hours of content in unit 3.1, and from what I can tell it only scratches the surface of React. I feel that a strong grasp of a front-end framework is going to be more valuable to students looking for their first junior role than some of the other content to which more time is devoted in the current roadmap (e.g. +20 hours mockups/wireframes, 10 hours security). More advanced topics like hooks, routing, state management could all be touched on after unit 3.1.

### CODE REVIEW

File: Unit3/Module3.1/Mini-project3.1a/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Mini Project 3.1a: Add a Validation Message
+
+##### Credit to OpenClassrooms
+![Become](https://github.com/OCclassprojects/logo/blob/master/fav-icon.png?raw=true)
+
+### Mini Project Instructions
+
+1. Open the [project page on OpenClassrooms](https://openclassrooms.com/en/courses/4286486-build-web-apps-with-reactjs/4317901-practice-add-a-validation-message)
+
+2. Create a new branch and then new Pull Request and assign it to your coach for review.
```
Comment (alaurenz) [inline Unit3/Module3.1/Mini-project3.1a/README.md]: Here it is probably helpful to tell them what specific files they should be putting into the PR (referring to files that they would be building out according to OpenClassroom instructions).

File: Unit3/Module3.1/Mini-project3.1b/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Mini Project 3.1b: Edit Existing Products
+
+##### Credit to OpenClassrooms
+![Become](https://github.com/OCclassprojects/logo/blob/master/fav-icon.png?raw=true)
+
+### Mini Project Instructions
+
+1. Open the [project page on OpenClassrooms](https://openclassrooms.com/en/courses/4286486-build-web-apps-with-reactjs/4317921-practice-extend-the-form-functionality-to-edit-existing-products)
+
+2. Create a new branch and then new Pull Request and assign it to your coach for review.
```
Comment (alaurenz) [inline Unit3/Module3.1/Mini-project3.1b/README.md]: Again, same as above, lets tell them what specific files they should be putting into the PR.

File: Unit3/README.md [hunk 3]
```diff
@@ -11,7 +11,8 @@
 ### Mini-Projects
 
 * Included in OC course:
-   * Mini-Project 3.1: [Add a Validation Message](https://openclassrooms.com/en/courses/4286486-build-web-apps-with-reactjs/4317901-practice-add-a-validation-message) and [Edit Existing Products](https://openclassrooms.com/en/courses/4286486-build-web-apps-with-reactjs/4317921-practice-extend-the-form-functionality-to-edit-existing-products)
+   * Mini-Project 3.1a: [Add a Validation Message](https://openclassrooms.com/en/courses/4286486-build-web-apps-with-reactjs/4317901-practice-add-a-validation-message)
```
Comment (alaurenz) [inline Unit3/README.md]: Similar comment as in my Unit 1 PR, make sure the links point to the READMEs in a relative way.

**Review by alaurenz — CHANGES_REQUESTED** [review]

**Comment by raevangregson** [comment]
I think that this [React tutorial](https://reactjs.org/tutorial/tutorial.html) from the React documentations hits a few more concepts in depth - it should be easy to follow, since it has example code through out as well as the final code pen at the end.  

I can think of some mini projects that could easily be built off of this tutorial after the fact such as:

- Breaking out components into their own files and importing them and exporting them correctly

- Using local storage to persist our state if we do a hard refresh

**Review by raevangregson — APPROVED** [review]
It looks like you addressed adrians comments - going to approve and merge, and open a PR of my own.

**Pull request merged**